python/servo/post_build_commands.py

```diff
@@ -82,15 +82,14 @@ def run(self, params, release=False, dev=False, android=None, debug=False, debug
                 return
             script = [
                 "am force-stop com.mozilla.servo",
-                "echo servo >/sdcard/Android/data/com.mozilla.servo/files/android_params"
             ]
-            for param in params:
-                script += [
-                    "echo '%s' >>/sdcard/Android/data/com.mozilla.servo/files/android_params"
-                    % param.replace("'", "\\'")
-                ]
+            extra = ""
+            # FIXME: see params comment above. This is supposed to be parsed by servo's `from_cmdline_args`.
```

Comment:
Not supporting arguments other than the URL is a significant regression. I’m considering relying on `--webdriver` to run WPT.

Reply:
Done.